cppcache/include/geode/CqQuery.hpp

```diff
@@ -53,6 +53,8 @@ class Query;
  */
 class APACHE_GEODE_EXPORT CqQuery {
  public:
+  virtual ~CqQuery() = default;
```

Comment:
This will for sure change the ABI. I am not sure we can do this on the public headers until the next major.